src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +74,16 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test(expected = UnsupportedOperationException.class)
+    @Category({RequiresCouch.class, RequiresCloudantLocal.class})

```

Comment:
Done in f6eea47, but sadly this only had the effect of moving 1 test method annotation to the class.